python/pyiceberg/io/pyarrow.py

```diff
@@ -122,6 +126,12 @@
 ONE_MEGABYTE = 1024 * 1024
 BUFFER_SIZE = "buffer-size"
 ICEBERG_SCHEMA = b"iceberg.schema"
+FIELD_ID = "field_id"
+FIELD_DOC = "doc"
```

Comment:
Should this constant name match its contents? That's what we would normally do so that code is readable.